src/iocore/net/QUICNetVConnection.cc

```diff
@@ -180,6 +180,10 @@ QUICNetVConnection::state_handshake(int event, Event *data)
     break;
   }
 
+  if (this->closed != 1 && (quiche_conn_is_closed(this->_quiche_con) || quiche_conn_is_draining(this->_quiche_con))) {
```

Comment:
Not directly related to this change, but should `closed` be a boolean? It seems like this would read better that way.

Reply:
I wish it were that simple. There is a place that sets it to -1 in UnixNetVConnection.

Reply:
Ah, got it. Thanks.